subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -160,7 +160,21 @@ $ gradle  mAL:cT
 include::{snippetsPath}/tutorial/nameMatching/tests/nameMatching.out[]
 ----
 
-You can also use these abbreviations with the -x command-line option.
+You can also use these abbreviations with the `-x` command-line option.
+
+=== Tracing name expansion
+
+For complex projects, it might not be obvious if the correct tasks were executed. When using abbreviated names, a single typo can lead to the execution of different tasks.
```

Comment:
Suggestion: 
> it might not be obvious if the **intended** tasks were executed

and:
> a single typo can lead to the execution of **unexpected** tasks.